content/desktop/wsl/use-wsl.md

```diff
@@ -21,5 +21,5 @@ The following section describes how to start developing your applications using
 
     This opens a new VS Code window connected remotely to your default Linux distro which you can check in the bottom corner of the screen.
 
-    Alternatively, you can type the name of your default Linux distro in your Start menu, open it, and then run `code` .
+    Alternatively, you can type the name of your default Linux distro in your Start menu, open it, and then run `code .`
```

Comment:
suggestion: `code .` was probably intentional here because it would open VS Code in the current working directory. But that could be clarified/worded better - and we don't really mention the current working directory implications here. I assume opening WSL either with `wsl` or from the start menu would put you in the WSL home directory? But that's not a very useful workspace to launch your editor in. You'd probably want to `cd` to the project directory first (inside wsl)